examples/compute-management/02-VirtualMachineCreation.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+printf "\n=== Managing Virtual Machine Creation in Azure Compute ===\n"
+
+printf "\n1. Creating a new resource group: %s and location: %s.\n" "$groupName" "$location"
+azure group create -n "$groupName" --location "$location"
+
+printf "\n2. Creating a new storage account '%s' in type '%s'.\n" "$storageAccountName" "$storageAccountType"
+azure storage account new --resourcegroupname "$groupName" --name "$storageAccountName" --location "$location" --type "$storageAccountType"
+
+printf "\n3. Create virtual network.\n"
+result=`azure virtual network create --resourcegroupname "$groupName" --name test --location "$location" --addressprefix "[\"10.0.0.0/16\"]" --subnet "[{\"Name\":\"test\",\"AddressPrefix\":\"10.0.0.0/24\"}]" --force`
+
+contextResult=`azure context get`
+
+subId=`echo $contextResult | jq '.Subscription.SubscriptionId' --raw-output`
+
+subnetId="/subscriptions/$subId/resourceGroups/$groupName/providers/Microsoft.Network/virtualNetworks/test/subnets/test"
+
+echo -n "$subnetId" > "$BASEDIR/$groupName.subnetIdFile"
+
+printf "\n4. Create network interface with:\r\nsubId='%s' \r\n& \r\nsubnetId='$subnetId'.\n" "$subId"
+azure network interface create --name test --resourcegroupname "$groupName" --location "$location" --subnetid @"$BASEDIR/$groupName.subnetIdFile"
```

Comment:
@markcowl @stankovski 
```--subnetid /subscription/xxx``` would result in ```c:\Program Files\Git\subscriptions...``` and thus fail the request in the tests.

```--subnetid @file``` would pass.

Why?